Makefile

```diff
@@ -415,3 +422,53 @@ docs-sync:
 	SRC=app/vmctl/README.md DST=docs/vmctl.md ORDER=8 $(MAKE) copy-docs
 	SRC=app/vmgateway/README.md DST=docs/vmgateway.md ORDER=9 $(MAKE) copy-docs
 	SRC=app/vmbackupmanager/README.md DST=docs/vmbackupmanager.md ORDER=10 $(MAKE) copy-docs
+
+enterprise-remote-check:
+ifeq ($(ENTERPRISE_REMOTE_EXISTS), 0)
+	$(error enterprise remote is missing)
+endif
+
+create-tags: enterprise-remote-check
+	git fetch --all
+	git switch enterprise && git tag -s $(TAG)-enterprise -m "$(TAG)"
+	git switch enterprise-cluster && git tag -s $(TAG)-enterprise-cluster -m "$(TAG)"
+	git switch cluster && git tag -s $(TAG)-cluster -m "$(TAG)"
+	git switch master && git tag -s $(TAG) -m "$(TAG)"
+
+push-tags: enterprise-remote-check
+	git push origin $(TAG)
+	git push origin $(TAG)-cluster
+	git push enterprise $(TAG)-enterprise
+	git push enterprise $(TAG)-enterprise-cluster
+
+github-token-check:
+ifndef GITHUB_TOKEN
+	$(error GITHUB_TOKEN is undefined)
+endif
+
+github-create-release: github-token-check push-tags
+	@release_id=$$(curl -s -w "%{http_code}" \
```

Comment:
It would be great if this Makefile rule could be called multiple times in the case of failures during creating of new release or failures during uploading the assets. Subsequent calls of `make github-create-release` should gracefully handle already created release and partially uploaded files. Probably, add `make github-delete-release` rule for cleaning up incompletely created release and/or incompletely uploaded assets?

Reply:
i was thinking about it, but also a bit hesitant about this as release deletion is destructive operation. And also there is no way to identify which release to delete as name is not unique. unless we will be preserving release IDs somewhere

As of now when you call rule multiple times, additional release drafts are created.